acctz/acctz.proto

```diff
@@ -94,28 +94,90 @@ message SessionInfo {
   // - multiple sessions over a single SSH connection
   // - multiple channels over a single gRPC connection
   string channel_id = 6;
+
+  // optional name of the tty, eg: console0, aux0, vty0, pty0.  If the
+  // connection is a serial line, such as a console, the adddress,
+  // port, and protocol fields above will be omitted.
+  string tty = 7;
+
+  // session status
+  enum SessionStatus {
+    SESSION_STATUS_UNSPECIFIED = 0;
+    SESSION_STATUS_LOGIN = 1;	// "start"
+    SESSION_STATUS_LOGOUT = 2;	// "stop"
+    SESSION_STATUS_ONCE = 3;	// simultaneous login, cmd, logout
+    SESSION_STATUS_ENABLE = 4;	// change privilege level
+    SESSION_STATUS_IDLE = 5;	// "watchog"
```

Comment:
'watchdog' ?

Reply:
This is what tacacs+ (rfc8907) calls this update message.  ie: "still logged-in".  There probably is a better term than "IDLE"; if you have a suggestion, please share/change it.

Reply:
I think that we are still waiting for comments from baires on these changes.  I suppose that should timeout at some point.

Reply:
@earies Do you have comments on this?  The window is closing on November 2.

Reply:
Hey @earies ,  I am resolving this so that we can move forward. Please open an issue or a PR to address whatever you would like to see changed.